src/main/markdown/doc/latest/tutorial/i18n.md

```diff
@@ -146,14 +147,14 @@ When you internationalize your application's interface, keep in mind that the la
 3.  Add the mappings for the static text in the German user interface.
     *  Copy and paste the following text into the StockWatcherConstant_de.properties file.
 
-```
-stockWatcher = Aktienbeobachter
-symbol = Symbol
-price = Kurs
-change = &Auml;nderung
-remove = Entfernen
-add = Hinzuf&uuml;gen
-```
+    ```
```

Comment:
I adjust this consequent to the other elements of the page. Line indenting in markdown is a pain. Adding more whitespaces will not have an effect. In my opinion this is the best solution.